src/mmrelay/migrate.py

```
@@ -0,0 +1,1858 @@
+                    logger.warning("Failed to backup GPX file: %s", e)
+
+            try:
+                if move:
+                    logger.info("Moving GPX file: %s", gpx_file)
+                    shutil.move(str(gpx_file), str(dest_path))
+                else:
+                    logger.info("Copying GPX file: %s", gpx_file)
+                    shutil.copy2(str(gpx_file), str(dest_path))
+                logger.debug("Migrated GPX file: %s", gpx_file)
+                migrated_count += 1
+            except (OSError, IOError) as exc:
```

Ruff (0.14.14)

[error] Local variable `exc` is assigned to but never used

Remove assignment to unused variable `exc`

(F841)